### PR TITLE
add Databricks configuration documentation

### PR DESCRIPTION
**Pull request by SirOibaf**

### CODE REVIEW

**Comment by SirOibaf** [comment]
I need to add some images.

File: docs/integrations/databricks.md [hunk 1]
```diff
@@ -1,3 +1,71 @@
...
+
+Users can get a valid Databricks API Key by following the [Databricks Documentation](https://docs.databricks.com/dev-tools/api/latest/authentication.html#generate-a-personal-access-token)
+
+!!! warning "Cluster access control"
+
+    If users have enabled [Databricks Cluster access control](https://docs.databricks.com/security/access-control/cluster-acl.html#cluster-access-control), it is important that the users running the cluster configuration (i.e. the user generating the API Key) has `Can Manage` privileges on the cluster they are trying to configure.
+
+## Register a new Databricks Instance
+
+Users can register a new Databricks instance by navigating the the `Integrations` tab of a project Feature Store. Registering a Databricks instance requires adding a the instance address and the API Key.
```
Comment (moritzmeister) [inline docs/integrations/databricks.md:23]:
```suggestion
Users can register a new Databricks instance by navigating to the `Integrations` tab of a project Feature Store. Registering a Databricks instance requires adding the instance address and the API Key.
```

File: docs/integrations/databricks.md [hunk 2]
```diff
@@ -1,3 +1,71 @@
...
+
+The instance address should be in the format `[UUID].cloud.databricks.com` (or `adb-[UUID].19.azuredatabricks.net` for Databricks on Azure), essentially the same web 
+address used to reach the Databricks instance from the browser.
+
+The API Key will be stored in the Hopsworks secret store for the user and will be available only for that user.  If multiple users need to configure Databricks clusters, each has to generate an API Key and register an instance. The Databricks instance registration does not have a project scope, meaning that once register, the user can configure clusters for all projects they are part of.
```
Comment (moritzmeister) [inline docs/integrations/databricks.md:28]:
```suggestion
The API Key will be stored in the Hopsworks secret store for the user and will be available only for that user.  If multiple users need to configure Databricks clusters, each has to generate an API Key and register an instance. The Databricks instance registration does not have a project scope, meaning that once registered, the user can configure clusters for all projects they are part of.
```

File: docs/integrations/databricks.md [hunk 3]
```diff
@@ -1,3 +1,71 @@
...
+    Currently Runtime 6 is suggested to be able to use the full suite of Hopsworks Feature Store capabilities. 
+
+## Configure a cluster
+
+Clusters are configured for a project user, which, in Hopsworks terms, means a user operating within the scope of a project. 
+To configure a cluster, click on the `Configure` button. By default the cluster will be configured for the user making the request. If the user doesn't have `Can Manage` privilege on the cluster, they can ask a project `Data Owner` to configure it for them. Hopsworks `Data Owners` are allowed to configure clusters for other project users, as long as they have the required Databricks privileges.
+
+During the cluster configuration the following steps will be taken:
+
+- Upload to DBFS an archive containing the necessary Jars for HSFS and HopsFS to be able to read and write from the Hopsworks Feature Store
```
Comment (moritzmeister) [inline docs/integrations/databricks.md:45]:
```suggestion
- Upload an archive to DBFS containing the necessary Jars for HSFS and HopsFS to be able to read and write from the Hopsworks Feature Store
```

**Review by moritzmeister — CHANGES_REQUESTED** [review]
few typos and grammar changes

**Review by moritzmeister — APPROVED** [review]

**Pull request merged**